src/allotropy/parsers/perkin_elmer_envision/perkin_elmer_envision_structure.py

```diff
@@ -169,16 +169,18 @@ def create(reader: CsvReader) -> BasicAssayInfo:
 
 @dataclass
 class PlateType:
-    number_of_wells: Optional[float] = None
+    number_of_wells: float
 
     @staticmethod
     def create(reader: CsvReader) -> PlateType:
         reader.drop_until("^Plate type")
         data = reader.pop_csv_block_as_df("^Plate type")
         data_df = assert_not_none(data, "Plate type").T
         return PlateType(
-            try_float_or_none(
-                str(df_to_series(data_df).get("Number of the wells in the plate"))
+            float(
```

Comment:
qq: why not `try_float` here?

Reply:
good question, `number_of_wells` is required in the ASM now I think, but as far as I can tell this line always exists?